tools/topology/sof-cml-rt5682.m4

```diff
@@ -88,11 +96,19 @@ PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
 
 # Passthrough capture pipeline 7 on PCM 5 using max 2 channels.
 # Schedule 1000us deadline on core 0 with priority 0
+ifdef(`DMIC16KPROC_FILTER1', `define(PIPELINE_FILTER1, DMIC16KPROC_FILTER1)', `undefine(`PIPELINE_FILTER1')')
+ifdef(`DMIC16KPROC_FILTER2', `define(PIPELINE_FILTER2, DMIC16KPROC_FILTER2)', `undefine(`PIPELINE_FILTER2')')
+define(`PGA_NAME', Dmic1)
```

Comment:
@singalsu the change looks fine to me when I winder why we decided to name the PGA widgets as DMIC0 and DMIC1 for this topology at all? we should just remove the hard coded PGA names from pipe-eq-iir-volume-capture and things should be back to normal isnt it?

Reply:
This preserves the sof-cml-rt5682 based topologies similar to git master. I'm not sure if the mic mute key/LED requirement applies to this platform but at least this way this patch does not break it (if it is needed).

There were two other topologies where I'm sure the there's no mic mute key/LED requirement so I didn't force for them Dmic0 and Dmic1 names: sof-apl-eq-dmic.tplg and sof-glk-eq-da7218.tplg. In fact those topologies can be later later cleaned up to use intel-generic-dmic.tplg.

Reply:
OK , lets leave it as is but the PGA widget names should affect the mute/LED requirement. That has to do with control name instead.

Reply:
@ranj063 the issue is that alsa is combining the user space control name from component+control. So the component name affects the mixer name you see in user space.